### DISCUSSION TITLE
Java profiling option missing

### DISCUSSION THREAD

**Discussion by kuikiker** [apache/skywalking#10149] [Q&A] [answered]
Hello, before upgrading we were able to profile specific endpoints using the old UI but after we upgraded skywalking we are unable to locate the (java) profile option.

Previous configuration:
- Skywalking: v8.11 (**elasticsearch** storage)
- Agent: v8.9

Current configuration:
- Skywalking: v9.2 (**mysql** storage)
- Agent: v8.12

Is there anything else we need to configure now? Thanks !

**Comment by wu-sheng** [comment]
It is in general service part still.

**Comment by kuikiker** [comment, marked as answer]
That is the issue I think, shouldn't the Profile option be there?

![2022-12-12_12h24_48](https://user-images.githubusercontent.com/3798929/207033693-710f91cd-4d14-4369-a672-4178085eb57d.png)

**Reply by wu-sheng** [reply]
No. The SDK profiling is service oriented.
There doesn't have to put all things on the home dashboard.
You don't need profiling everyday, it only diagnosises service bottlenecks when needed. right?

**Reply by kuikiker** [reply]
Sure, we only need to sporadically profile certain endpoints to analyse bottlenecks. But how do we profile now ? I mean, before we were able to setup through the UI a profling/sampling scenario as shown here:

<img width="1676" alt="profiling-create" src="https://user-images.githubusercontent.com/3798929/207042222-7ed60d57-e581-4ae4-a436-9942c0e91da6.png">

**Reply by wu-sheng** [reply]
This is where it is now.

<img width="1919" alt="image" src="https://user-images.githubusercontent.com/5441976/207042693-9df2fb27-b8b3-48db-b679-527cc01fc39c.png">

**Reply by kuikiker** [reply]
Thanks ! This is what I needed.
Sorry for the misunderstanding.
